ui/src/app/settings/containers/Storage/Storage.js

```diff
@@ -42,76 +69,70 @@ const Storage = () => {
       {loaded && (
         <Formik
           initialValues={{
-            defaultStorageLayout,
-            enableDiskErasing,
-            diskEraseWithQuick,
-            diskEraseWithSecure
+            default_storage_layout: defaultStorageLayout,
+            enable_disk_erasing_on_release: enableDiskErasing,
+            disk_erase_with_quick_erase: diskEraseWithQuick,
+            disk_erase_with_secure_erase: diskEraseWithSecure
           }}
-          onChange
-          onSubmit={(values, { setSubmitting }) => {
-            setTimeout(() => {
-              alert(JSON.stringify(values, null, 2));
-              setSubmitting(false);
-            }, 400);
+          onSubmit={values => {
+            Object.keys(values).forEach(key => {
+              dispatch(saveConfig(key, values[key]));
+            });
           }}
         >
-          {({
-            handleBlur,
-            handleChange,
-            handleSubmit,
-            isSubmitting,
-            values
-          }) => (
+          {({ handleBlur, handleChange, handleSubmit, touched, values }) => (
             <Form onSubmit={handleSubmit}>
               <div className="row">
                 <div className="col-8">
                   <Select
```

Comment:
These can use the new Formik field component, see: https://github.com/canonical-web-and-design/maas-ui/pull/62#discussion_r312314179